.github/workflows/beam_PostCommit_Python_Examples_Flink.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       github.event_name == 'workflow_dispatch' ||
       startsWith(github.event.comment.body, 'Run Python Examples_Flink')
     runs-on: [self-hosted, ubuntu-20.04, main]
-    timeout-minutes: 120
+    timeout-minutes: 240
```

Comment:
if it takes so long might due to some other issue, the same test on Jenkins takes short time to finish: https://ci-beam.apache.org/view/PostCommit/job/beam_PostCommit_Python_Examples_Flink/buildTimeTrend